asciidoctorj-core/src/main/java/org/asciidoctor/jruby/cli/AsciidoctorInvoker.java

```diff
@@ -186,8 +182,8 @@ private String renderInput(Asciidoctor asciidoctor, Options options, List<File>
     }
 
     private String readInputFromStdIn() {
-        Scanner in = new Scanner(System.in);
-        String content = in.nextLine();
+        Scanner in = new Scanner(System.in).useDelimiter("\\A");
+        String content = in.next();
```

Comment:
I see, this code was not correct before and only read the first line of the input from stdin (if it would have been called, which it wasn't yet)

Can you please call IOUtils.readFull() here instead of copying its implementation here?

(I am actually not too happy about using Scanner for this and would like to replace that now that I have seen it. Scanner doesn't seem to be the right thing for reading a file fully into memory)